torch/csrc/jit/interpreter.cpp

```diff
@@ -717,6 +720,24 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
     return false;
   }
 
+  void throwError(const std::string& error_msg, bool is_jit_exception) {
```

Comment:
I think we should try to clean up this exception handling logic. There are too many potential combinations (rethrow, JITException, wrapped exception that may or may not be a JITException.  I think for now it is fine to turn every exception into a JITException.